records/2022-Q4-activity-report.md

```diff
@@ -0,0 +1,420 @@
+---
+layout: contact
+title: Scala Center Activity Report for 2022 Q4
+---
+
+Scala Center team: Quentin Bernet, 100%; Sébastien Doeraene, 100%; Darja Jovanovic, 100%; Anatolii Kmetiuk, 100%;
+Guillaume Martres, 100%; Adrien Piquerez, 100%; Julien Richard-Foy, 80%; Jamie Thompson, 100%.
+VirtusLab team: Jędrzej Rochala, 100%. Lunatech team: Chris Kipp, 33%.
+
+External contractors, part-time: Adam Goodman, leadership and
+governance expert.
+
+## At a Glance
+{: .no_toc}
+
+* Table of Contents
+{:toc}
+
+## Tooling
+
+### Debugger in Metals
+
+For Scala 2 and Scala 3.
+
+We shipped a new major version of the Scala Debug Adapter in Metals that improves the debugging experience in Scala 2 and Scala 3. In this new version, we improved the debugging steps, we added support for conditional breakpoints, logpoints and more features. See a full list of features in the release notes of the Scala Debug Adapter [3.0.1](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.1) and [3.0.3](https://github.com/scalacenter/scala-debug-adapter/releases/tag/v3.0.3).
+
+### BSP defaults for Metals
+
+An area of improvement we've identified with Metals and the usage of the Build
+Server Protocol (BSP) in general is that it provides many ways to get started
+with a Scala project. We'd like to minimize this and revisit the default choices
+that Metals makes of using Bloop no matter the build tool. The aim is to make
+the experience a newcomer has smoother, abstracting away things like Bloop and
+BSP, while still allowing for a powerful user experience for advanced users.
+We're still in the experiment and research stage for this which is leading to
+creating various issues around the ecosystem. However, the discussion and main
+work can be tracked in [metals#4505](https://github.com/scalameta/metals/discussions/4505#discussioncomment-4571546).
+
+### sbt
+
+For Scala 2 and Scala 3
+
+The Maven coordinates of sbt plugins are invalid based on the Maven specifications, as described in [sbt#3410](https://github.com/sbt/sbt/issues/3410)). This is causing a number of issues:
+- Maven cannot resolve sbt plugin artifacts.
+- Other resolution tools, like Coursier, must implement some sbt-specific hack to resolve them.
+- Some Maven proxies cannot check the validity of sbt plugins and they reject them.
+- The number of downloads of sbt plugins are unavailable on [oss.sonatype.org](https://oss.sonatype.org/).
+- [javadoc.io](https://javadoc.io/) cannot find the scaladoc of sbt plugins.
+- And more
+
+We implemented a solution that enables a smooth transition to valid Maven pattern of sbt plugins in [sbt#7096](https://github.com/sbt/sbt/pull/7096). This solution will be available in sbt 1.9.x, with bidirectional compatibility: sbt 1.9.x will publish the valid and legacy artifacts of sbt plugins, and will be able to resolve the valid or legacy artifacts of sbt plugins.
+
+### Language Improvements
+
+For Scala 3.
```

Comment:
I find it a bit odd that these are just floating here like this. Would it be better to maybe put them in the headers? Like instead of having this here in the headers put like (_For Scala 3_), or even put this as a subscript instead?

Reply:
I agree with your observation, but I am not sure which alternative solution would be better.

If you are interested, feel free to make some experiments.